vendor/bat-native-ledger/src/bat/ledger/internal/state/state_migration.cc

```diff
@@ -25,12 +25,27 @@ StateMigration::StateMigration(LedgerImpl* ledger) :
     v3_(std::make_unique<StateMigrationV3>()),
     v4_(std::make_unique<StateMigrationV4>(ledger)),
     v5_(std::make_unique<StateMigrationV5>(ledger)),
+    v6_(std::make_unique<StateMigrationV6>(ledger)),
     ledger_(ledger) {
-  DCHECK(v1_ && v2_ && v3_ && v4_ && v5_);
+  DCHECK(v1_ && v2_ && v3_ && v4_ && v5_ && v6_);
 }
 
 StateMigration::~StateMigration() = default;
 
+void StateMigration::Start(ledger::ResultCallback callback) {
+  const int current_version = ledger_->state()->GetVersion();
+  const bool fresh_install = current_version == 0;
+
+  if (fresh_install) {
+    BLOG(1, " Fresh install, state version set to " << kCurrentVersionNumber);
```

Comment:
Nit: Extra space at beginning of message.

Reply:
fixed